include/insilico/core/engine/data.hpp

```diff
@@ -35,6 +35,18 @@
 namespace insilico {
 namespace engine {
 
+
+
+bool time_specified;
```

Comment:
There should be an explicit mention of a default value here. In this case, I suppose, it should be `false`.